en/docs/learn/configuring-outbound-provisioning.md

```diff
@@ -0,0 +1,180 @@
+# Configuring Outbound Provisioning 
+
+## Introduction
+
+Provisioning is the process of creating, maintaining and deleting digital identities (accounts) for users of a system(s) and linking appropriate rights to identities in the form of rules and roles. 
+
+This section of the tutorial demonstrates configuring Salesforce as the identity provider to provision users from WSO2 Identity Server. This means that once this is configured, new users that are added to WSO2 Identity Server are added to Salesforce as well. 
+
+## Scenario
+
+To demonstrate the scenario in this tutorial, let's assume a taxi company called "Pickup" are expanding and they are about to hire a new sales team. The drivers employed at Pickup use an application called **Pickup Dispatch** for managing their taxi orders and daily operations. Pickup maintains its employee user accounts using WSO2 Identity Server and has already set up outbound provisioning between WSO2 Identity Server and Pickup Dispatch for the drivers. The new sales team will need to access **Pickup Dispatch** as well but they will also need to use a **Salesforce** application to manage their daily sales activities.
```

Comment:
let's assume that a taxi company called "Pickup" is expanding and is about to hire a new sales team.

to  manage their